content.test.ts

```diff
@@ -268,6 +280,63 @@ test("test streaming content", { timeout: 180000 }, async () => {
   expect(refinedContent.title).toBe(title);
   expect(refinedContent.version).toBe(1);
   expect(refinedContent.commands.length).toBe(2);
+  expect(refinedContent.status).toBe("DRAFT");
+  expect(refinedContent.publishedVersion).toBe(undefined);
 
   await catalog.delete();
 });
+
+test(`test content status and publishing`, { timeout: 180000 }, async () => {
+  testClient.configureOrg({
+    companyName: "Cortex Click",
+    companyInfo:
+      "Cortex Click provides an AI platform for go-to-market. Cortex click allows you to index your enterprise knowledge base, and create agents called Cortexes that automate sales and marketing processes like SEO, content writing, RFP generation, customer support, sales document genearation such as security questionairres and more.",
+    personality: [
+      "friendly and helpful",
+      "expert sales and marketing professional",
+      "experienced software developer",
+    ],
+    rules: [
+      "never say anything disparaging about AI or LLMs",
+      "do not offer discounts",
+    ],
+  });
+
+  const cortex = await testClient.configureCortex(
+    `cortex-${Math.floor(Math.random() * 10000)}`,
+    {
+      friendlyName: "Cortex AI",
+      instructions: ["answer questions about the cortex click AI GTM platform"],
+      public: true,
+    },
+  );
+
+  // create content
+  const title = "Overview of the Cortex Click AI GTM Platform";
+  const prompt =
+    "Write a blog post about the Cortex Click AI GTM Platform. Elaborate on scenarios, customers, and appropriate verticals. Make sure to mention the impact that AI can have on sales and marketing teams.";
+  const content = await cortex.generateContent({ title, prompt });
+
+  expect(content.status).toBe("DRAFT");
+
+  await content.setStatus("IN_REVIEW");
```

Comment:
Any reason for `IN_REVIEW` vs just `REVIEW`?

Reply:
I started with REVIEW, and it nagged my brain a bit. IN_REVIEW is an adjective (right? ChatGPT fix my terms), but REVIEW is a verb.

Just seemed more correct. Also easy to change.

After that huge discussion we had, I also realized we could simplify and just have DRAFT and PUBLISHED for the MVP, and consider this life cycle down the road. I think my "shed complexity" mode is still warming up....

Reply:
That makes sense. 

I'm less opinionated if we expose a friendly enum. I think my concern here is more about forcing users to type out the full string that includes caps and underscore. Fine to leave as is if we add the enum.